src/pandablocks_ioc/ioc.py

```diff
@@ -1150,7 +1072,8 @@ def _make_pos_mux(
 
         # This should be an mbbOut record, but there are too many posssible labels
         # TODO: There will need to be some mechanism to retrieve the labels,
-        # but there's a POSITIONS table that can probably be used
+        # but there's a POSITIONS table that can probably be used.
+        # OR PVAccess somehow?
```

Comment:
I believe so unless someone else has handled it in fixes.

@coretl  and I agree that given the many TODO's and the [issues coming in](https://github.com/PandABlocks/PandABlocks-ioc/pull/27#issuecomment-1705277035), it would make more sense to merge the progress made so far (since the changes have already been reviewed in their own PRs), then make new PRs for the changes to be made from TODOs.